vite.config.ts

```diff
@@ -0,0 +1,15 @@
+/// <reference types="vitest" />
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+  test: {
+    globals: true,
+    environment: 'jsdom',
+    setupFiles: ['./__tests__/vitest.setup.ts'],
+    include: ['**/__tests__/**/*.test.ts'],
+    coverage: {
+      include: ['src/**/*.ts', '!src/**/*.{d,cjs}.ts'],
+      reportsDirectory: 'coverage'
+    }
```

Comment:
I think that to to fix the coverage on CI we need to install `c8` as dev dependency and change the coverage setup to:

```suggestion
    coverage: {
      include: ['src/**/*.ts', '!src/**/*.{d,cjs}.ts'],
      reportsDirectory: 'coverage',
      enabled: true,
      reporter: ['text', 'lcov', 'json'],
    }
```